site/docs/master/customize-installation.md

```diff
@@ -42,6 +45,39 @@ By default, `velero install` does not install Velero's [restic integration][3].
 
 If you've already run `velero install` without the `--use-restic` flag, you can run the same command again, including the `--use-restic` flag, to add the restic integration to your existing install.
 
+## Enable features
+
+New features in Velero will be released as beta features behind feature flags which are not enabled by default. For a full listing of available feature flags, please refer `pkg/features/features.go`.
```

Comment:
it'd be nice to include a link to `features.go` here.

Reply:
My bad. That's not a valid file. Our feature flag constants are not limited to a single file. Going to link our constants file here instead of the feature code.

Reply:
I don't like linking to the Go code, mostly because in general I hate making people read code to understand a project. That being said, I also hate duplicating this kind of stuff, so I don't have a great alternative to generating docs based on the code right now.

My 2c - we keep the number of feature flags limited and be aggressive about deprecating them when the time is right so we don't have to worry about a crazy long list. If they're defined on an installation or in a client's config but no longer used, that should be a no-op.

Reply:
:+1: it might be helpful in the future to move them to their own dedicated file, for clarity, but we can do that separately.

Reply:
+1 for keeping the feature flag list small.
But we should also have a way for users to discover feature flags. That's the problem I am looking to solve here and this was the easiest way to do that today.
Some options for us to consider:
1. having a features page on the website
2. having a CLI command list all features

My vote is for option2

Reply:
Yeah, for today, I think this is the best option.

Per the design I wrote, I explicitly opted not to have a fully enumerated list of features. We can revisit that if it's easier, and I think Kubernetes does this. For now, we can log future action IMO.

Reply:
Created https://github.com/vmware-tanzu/velero/issues/2515